tests/services/test_telegram_persona_pipeline.py

```
@@ -0,0 +1,62 @@
+from pathlib import Path
+from types import SimpleNamespace
+from unittest.mock import AsyncMock, patch
+
+import pytest
+
+os.environ.setdefault("SECRET_KEY", "test-secret")
+os.environ.setdefault("DATABASE_URL", "sqlite+aiosqlite:///./test.db")
+
+sys.path.append(str(Path(__file__).resolve().parents[2]))
+
+from app.api.v1.endpoints.telegram import _process_natural_language  # noqa: E402
```

Ruff (0.13.2)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)